README.md

```diff
@@ -116,4 +117,13 @@ This will not remove the volumes on harbor side, to do so you'll have to stop ha
 And you'll need to delete the data directory (sudo is needed due to files created/modified inside the containers):
 - `sudo rm -rf .harbor/harbor/data`
 
+
+### Notes on setting up Harbor in production
+
+In development, every component of Harbor is deployed via docker-compose. In production, we use an external database. Currently, only PostgreSQL is supported by Harbor. This PostgreSQL db should be set up as a Trove instance with a Cinder volume.
+
+To configure Harbor for use with an external db, you need to uncomment the `external_database` section in `harbor.yml` and fill in the necessary information. For the config changes to take effect, you need to run the `./install.sh` afterwards script. Remember to also remove the db service from Harbor's `docker-compose.yml`.
```

Comment:
Can you use the path from the root of the repo for the files you mention? (install.sh, harbor.yaml)
Or alternatively, mention that you have to 'cd .harbor/harbor' before or similar.

Reply:
Added the full path.